src/main/scala/dpla/ingestion3/messages/IngestMessageTemplates.scala

```diff
@@ -56,6 +56,15 @@ trait IngestMessageTemplates {
       value = "both rights and edmRights are defined"
     )
 
+  def duplicateOriginalId(id: String): IngestMessage =
+    IngestMessage(
+      message = "Duplicate",
+      level = IngestLogLevel.warn,
```

Comment:
My only suggestion is to change this from warning to error so that we fail all duplicate records rather than indexing one of them.

Reply:
Will do, thanks @moltude